pennylane/__init__.py

```diff
@@ -133,6 +136,63 @@ def device(name, *args, **kwargs):
     )
 
 
+def load(quantum_circuit_object, name: str):
+    """load(name, quantum_circuit_object)
+    Load a plugin :func:`~.load` which can then be used to convert objects of
+    quantum circuits from other frameworks.
+
```

Comment:
This is on my todo list, but lets start adding a minimal example to each docstring showing how the function is used; similar to how the GBS guys have been doing it in Strawberry Fields, or how NumPy does it in their documentation:
```suggestion

**Example:**

>>> qc = qiskit.QuantumCircuit(2)
>>> qc.rz(0.543, [0])
>>> qc.cx(0, 1)
>>> my_circuit = qml.load(qc, name='qiskit')
>>>
>>> @qml.qnode(dev)
>>> def circuit(x):
>>>     qml.RX(x, wires=1)
>>>     my_circuit(wires=(1, 0))
>>>     return qml.expval(qml.PauliZ(0))
```

This is important now that the docs have been restructured, as each function has it's own page, and there might not be enough context for how each function works.

Reply:
Note that within docstrings, the example should be as short and small as possible in order to show how the function is used.